File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/SimplePark.java
package org.firstinspires.ftc.teamcode;

import com.qualcomm.robotcore.eventloop.opmode.Autonomous;

import org.firstinspires.ftc.teamcode.implementations.Auto;
import org.firstinspires.ftc.teamcode.implementations.DriveTrain;

@Autonomous(group = "Auto", name = "SimplePark")
public class SimplePark extends Auto {
    @Override
    public void run() {
        // park
        bot.driveTrain.goAngle(12, DriveTrain.BUILDING_ZONE, 0.5);
    }
}

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/SimplePark.kt
package org.firstinspires.ftc.teamcode

import com.qualcomm.robotcore.eventloop.opmode.Autonomous
import org.firstinspires.ftc.teamcode.implementations.Auto
import org.firstinspires.ftc.teamcode.implementations.DriveTrain

@Autonomous(group = "Auto", name = "SimplePark")
class SimplePark : Auto() {
    override fun run() { // park
        bot.driveTrain.goAngle(12.0, DriveTrain.LOADING_ZONE, 0.5)
    }
}